lte/gateway/c/oai/tasks/mme_app/mme_config.c

```diff
@@ -795,8 +795,7 @@ int mme_config_parse_file(mme_config_t *config_pP)
     config_pP->gummei.nb = 0;
     if (setting != NULL) {
       num = config_setting_length(setting);
-      AssertFatal(
-        num == 1, "Only one GUMMEI supported for this version of MME");
+      OAILOG_INFO(LOG_MME_APP, "Number of GUMMEIs supported =%d\n", num);
```

Comment:
What is the side effect of ignoring the absence of GUMMEI?

Reply:
yes right, absence of GUMMEI effects the attach procedure. 
while forming the TAI list IE in attach accept message, we consider the GUMMEI list and serving plmn. In case of absence of GUMMEI, TAI list(Mandatory IE) will be sent with '0' value and attach procedure will not be completed.
Hence ERROR check is required in case of absence of GUMMEI. 
Will update the PR with appropriate changes.

Reply:
Added condition to validate at least one GUMMEI and at most 5 GUMMEIs are configured